### ISSUE TITLE
Interface Re-design

### ISSUE THREAD

**Issue by psgs** [RigPM/rigpm.github.io#1]
Recently, as I was browsing the Rig Package Manager website, I took the liberty of taking some time to  create some quick grey-scale mock-ups of interfaces that could be used in a potential re-design.
I was wondering if anybody would be able to take a look and make suggestions that would improve the designs, before I attempted to implement the designs in HTML and CSS.

Here is a potential design for the main page of the website:
![rig-package-manager](https://cloud.githubusercontent.com/assets/3946887/6210033/39f89b38-b617-11e4-8642-7928d3451a2b.jpg)

And here is a potential design for the documentation page:
![rig-documentation](https://cloud.githubusercontent.com/assets/3946887/6210039/493a8692-b617-11e4-93b8-2ca15c6f398c.jpg)

Thanks @deavmi for creating this wonderful project. I'm quite interested to see a Python based package manager that seamlessly :smiley:

**Comment by deavmi** [comment]
@psgs Yep. This time I won't noob out. But I think things will go well, I need to read the Python documentation more often, I think I write binaries in RIg (Possibly incorrectly), I am using the:

````Python
from urllib.request import urlopen
deb_writer = open("my_debfile.deb", "w").write(urlopen("http://google.com.debfile.deb").read())
````

Although I think I should being using the **wb** writing method for _Writing binaries_ rather than the **w** method for _Writing normal text and stuff_.

But I will see, will test it later this week, just fire up a Python terminal and do some I/O tests in Python.

**Comment by deavmi** [comment]
@psgs Yep. Simplicity in mind, just simple files, that have simple and easily parseable syntax to them.

**Comment by deavmi** [comment]
@psgs Bro .... That mockup of the site looks so beautiful.

**Comment by deavmi** [comment]
Infact I will mke a remind to check on I/O in Python.

**Comment by deavmi** [comment]
I will also be releasing v2.0 one day when I have time to work on it, it will be able to handle dependancies and post-installation or before-installation scripts. Which I think is quite neat.

**Comment by deavmi** [comment]
I am good with the initial design you psoted here! :smile: @psgs

**Comment by deavmi** [comment]
@psgs And I really do love that documentation page. This keep the documentation page on hold for now tho.

**Comment by psgs** [comment]
Hi @deavmi,

Thanks for your feedback! Dependencies and post/pre installation scripts sound nice :smiley: 
I have now produced a prototype of the home interface, and pushed it to this repository.

Thanks again,
psgs
